CHANGELOG.next.asciidoc

```diff
@@ -145,6 +145,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Fix Powershell processing of downgraded engine events. {pull}18966[18966]
 - Fix unprefixed fields in `fields.yml` for Powershell module {issue}18984[18984]
 - Remove top level `hash` property from sysmon events {pull}20653[20653]
+- Use ECS 1.7 ingress/egress instead of inbound/outbound network.direction in sysmon. {pull}22997[22997]
```

Comment:
I'm thinking this belongs under the breaking change section for Winlogbeat. WDYT?

Reply:
Oh it is there. I got confused about what section it was in since there are duplicate titles.